src/modules/users/mappers.ts

```diff
@@ -0,0 +1,11 @@
+import { UserModel } from './model';
+
+export const fromRawToUserData = (record: UserModel) => ({
+    id: record.id,
+    login: record.login,
+    age: record.age
+});
+
+export const fromRawToLoginSuggestResults = (records: UserModel[]) => ({
+    results: records.map((record) => record.login)
```

Comment:
mapper should not contain the response structure, only the data structure. This particular mapper should return just a simple array.

Reply:
done